src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -561,20 +561,255 @@ internal static int NonWindowsGetProcessParentPid(int pid)
         // the string perror would give you for the last set value of errno.
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-windows`
UnixFileSystem additions.Other properties of UnixStat object.Error occurred in Context block
```powershell
The term '/bin/ls' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\UnixStat.Tests.ps1:98 char:18
```